jiant/modules/seq2seq_decoder.py

```diff
@@ -44,8 +44,8 @@ def __init__(
 
         # We need the start symbol to provide as the input at the first timestep of decoding, and
         # end symbol as a way to indicate the end of the decoded sequence.
-        self._start_index = self.vocab.get_token_index(START_SYMBOL, self._target_namespace)
-        self._end_index = self.vocab.get_token_index(END_SYMBOL, self._target_namespace)
+        self._start_index = self.vocab.get_token_index("<SOS>", self._target_namespace)
```

Comment:
is this different than `START_SYMBOL` ?

Reply:
Yes.

Reply:
Can you search the codebase that this won't break somewhere else ? I'm worried that in preprocessing or in task definitions, we defined the token differently.

Reply:
Done.